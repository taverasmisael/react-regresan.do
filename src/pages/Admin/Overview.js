import React, { PureComponent } from 'react'

export default class OverviewPage extends PureComponent {
  render() {
    return (
      <React.Fragment>
        Overview
      </React.Fragment>
    )
  }
}
