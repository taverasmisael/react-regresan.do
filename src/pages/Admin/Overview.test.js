import React from 'react'
import { shallow } from 'enzyme'
import Overview from './Overview'

describe('routes', () => {
  describe('Pages/Admin/Overview', () => {
    it('renders without crashing', () => {
      shallow(<Overview />)
    })
    it('matches snapshot', () => {
      expect(shallow(<Overview />)).toMatchSnapshot()
    })
  })
})
