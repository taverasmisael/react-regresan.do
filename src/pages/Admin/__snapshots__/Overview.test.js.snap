// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`routes Pages/Admin/Overview matches snapshot 1`] = `
<React.Fragment>
  Overview
</React.Fragment>
`;
